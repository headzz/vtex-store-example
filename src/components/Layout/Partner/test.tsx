import { render } from '@testing-library/react'

import Partner from './Partner'

describe('<Partner />', () => {
  it('should render component that wraps partners section on page', () => {
    const { container } = render(<Partner />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
