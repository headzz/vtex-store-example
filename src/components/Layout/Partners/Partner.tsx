import { PartnerStyled } from './styled'

/**
 * @component
 * @name Partner
 * @description Defines the container for the layout of Partner.
 * @namespace components components/Layout/Partner
 * @param {element} children Component(s) that may appear inside the layout.
 *
 */

const Partner: React.FC = ({ children }) => {
  return <PartnerStyled>{children}</PartnerStyled>
}

export default Partner
