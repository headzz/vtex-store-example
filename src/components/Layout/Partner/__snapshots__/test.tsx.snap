// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[
  `<Partner /> should render component that wraps partners section on page 1`
] = `
.c0 {
  background-color: #F2F2F2;
  padding: 2.4rem 0;
  width: 100%;
}

<section
  class="c0"
/>
`
